backend/main.py

```diff
@@ -1,9 +1,10 @@
 from config.settings import settings
-from endpoints import members, messages, rooms, sync, threads
+from endpoints import members, messages, rooms, sync, threads, file
```

Comment:
the file name should always be in plural to communicate processing of multiple resource.
so rename `file.py` to `files.py`